model/task/db.go

```diff
@@ -164,6 +164,11 @@ func ByVersion(version string) db.Q {
 	})
 }
 
+// ByVersion produces a query that returns tasks for the given version.
+func ByVersions(versions []string) db.Q {
```

Comment:
I'd just return the results of the query in public functions... I know that evergreen's not great here. 

It's particularly bad in the task package because a lot of the find functions merge in task results, which we probably never want.